.github/actions/environment/action.yml

```diff
@@ -8,6 +8,11 @@ runs:
         echo TEST_TIMEOUT=310 | tee -a $GITHUB_ENV
         echo NO_OUTPUT_TIMEOUT=320 | tee -a $GITHUB_ENV
         echo PRESERVE_ENVVARS=REPLICATION_SYNC_TIMEOUT,TEST_TIMEOUT,NO_OUTPUT_TIMEOUT | tee -a $GITHUB_ENV
+        # For self-hosted runners run w/o restart may need to kill
+        # hanged processes that could be leaved from the previous
+        # workflows runs.
+        ps auxT | grep " tarantool " | awk '{system("kill -9 "$2)}' || true
+        ps auxT | grep "test-run.py " | awk '{system("kill -9 "$2)}' || true
```

Comment:
What for we need space after `test-run.py`?

Reply:
Just again to be sure the needed process is found, but not looked the same.